test/unit/test_ftd_configuration.py

```diff
@@ -93,10 +93,10 @@ def test_module_should_fail_when_check_mode_exception(self, resource_mock):
 
     def test_module_should_run_successful(self, resource_mock):
         operation_name = 'test name'
-        resource_mock.return_value = 'ok'
+        resource_mock.return_value = {'result': 'ok'}
```

Comment:
can we move fixes for tests to other patch ?

Reply:
Good point. Create a separate PR here: https://github.com/CiscoDevNet/FTDAnsible/pull/83